### PR TITLE
Explanations for canonicalization

### PR DESCRIPTION
**Pull request by iherman**
Added explanations to the canonicalization step and also some to the first processing part.

(This is a purely editorial change, no change on the technical content.)

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wpub/pull/380.html" title="Last updated on Dec 10, 2018, 11:52 AM GMT (56d853e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wpub/380/4b697d1...56d853e.html" title="Last updated on Dec 10, 2018, 11:52 AM GMT (56d853e)">Diff</a>

### CODE REVIEW

**Comment by iherman** [comment]
@mattgarrish if you think it is o.k., you can safely merge this...

File: index.html [hunk 1]
```diff
@@ -2974,134 +2983,367 @@ <h3>Obtaining a manifest</h3>
...
+			                            data-cite="!html#the-lang-and-xml:lang-attributes">set</a> to the value of
+			                            <var>l</var>, then add <br />
+			                        <code class="json">"name": [{"value": t, "language": l}]</code>
+			                    </li>
+			                    <li>or <br />
+			                        <code class="json">"name": [t]</code><br /> otherwise </li>
+			                </ul>
+			                <details>
+			                    <summary>Explanation</summary>
+			                    <p>This step ensures that the <code>title</code> element's content in the primary entry page can be used, as a default, as the title of the publication. I.e.,</p>
```
Comment (mattgarrish) [inline index.html]: Minor nit, but what follows are examples of what is being stated, not alternate expressions, so should lead with a "for example" not "i.e."

File: index.html [hunk 2]
```diff
@@ -2974,134 +2983,367 @@ <h3>Obtaining a manifest</h3>
...
+			        </li>
+			        <li> (<a href="#default-reading-order"></a>) if <var>manifest["readingOrder"]</var> is
+			                <code>undefined</code>, let <var>u</var> be the value of <a
+			                data-cite="!dom#concept-document-url">document.URL</a>, and add<br />
+			            <code class="json">"readingOrder": [{"type": ["LinkedResource"], "url": u}]</code><br /> to the
+			                <var>manifest</var>
+						<details>
+							<summary>Explanation</summary>
+							<p>
+								In case the publication consists of the primary entry page as an only content, the default reading order can be ommited; it will consist, automatically, of that single resource.
```
Comment (mattgarrish) [inline index.html]: "an only content" probably s/b "the only", but maybe rephrase this more simply as "If the publication consists only of the primary entry page"

Comment (mattgarrish) [reply index.html]: Also, s/ommited/omitted/

**Comment by iherman** [comment]
Thanks @mattgarrish. The usage of "I.e." was actually repeated elsewhere, I changed all of those.

Merging...
